src/Service/NotificationAndMailSender.php

```diff
@@ -159,6 +190,31 @@ private function getRecipientsAdmin(?Territory $territory): ArrayCollection
         return $recipients;
     }
 
+    private function getRecipientsPartners(bool $isFilteredAffectationStatus): array
+    {
+        $partnerRecipientsMail = new ArrayCollection();
+        $partnerRecipientsInAppNotif = new ArrayCollection();
+        foreach ($this->signalement->getAffectations() as $affectation) {
+            if (!$isFilteredAffectationStatus
+                    || AffectationStatus::STATUS_WAIT->value === $affectation->getStatut()
+                    || AffectationStatus::STATUS_ACCEPTED->value === $affectation->getStatut()) {
+                $partnerRecipientsMailItem = $this->getRecipientsPartner($affectation->getPartner());
+                $partnerRecipientsInAppNotifItem = $this->getRecipientsPartner($affectation->getPartner(), false);
```

Comment:
Plutôt que `$affectation->getPartner()`, utiliser ce qu'a fait @numew 
`$partner = $this->partnerRepository->getWithUserPartners($affectation->getPartner());`

dans https://github.com/MTES-MCT/histologe/pull/3713/files

Reply:
Ok, c'est fait ; ainsi que le rebase !